intern/remote/boss/remote.py

```diff
@@ -683,6 +683,21 @@ def list_coordinate_frames(self):
         cf = CoordinateFrameResource(name='')
         return self._list_resource(cf)
 
+    def get_channel(self, chan_name, coll_name, exp_name):
```

Comment:
Maybe for this helper the args should be in the logical order (collection, experiment, channel)? Is this useful or more confusing?